tree/dataframe/inc/ROOT/RDF/PyROOTHelpers.hxx

```diff
@@ -69,6 +70,31 @@ ROOT::RDF::RResultPtr<std::vector<T>> RDataFrameTake(ROOT::RDF::RNode df, std::s
    return df.Take<T>(column);
 }
 
+// MakeRDataFrame helpers
+
+// Dummy RResultPtr class so that we can reuse RLazyDS for reading numpy arrays
+template <typename T>
+class RResultPtrDummy {
```

Comment:
This type is not a `RResultPtr`:
* it does not trigger event loops
* it's missing most of the public interface of a `RResultPtr`
* `T` should only ever be `RVec`s
* it is meant to only be used in conjunction with `RLazyDS`

Maybe move it to `RLazyDS`'s header and/or rename it to `RVecPtr` or `RLazyDSColumnPtr`?

Reply:
I fully agree that this needs to be renamed!